cyences_app_for_splunk/default/data/ui/views/cs_o365_defender_atp_audit.xml

```diff
@@ -0,0 +1,76 @@
+<form theme="dark">
+  <init>
+    <set token="tkn_host">host="*"</set>
+    <set token="tkn_host_error_message">host="*"</set>
+  </init>
+  <label>Microsoft 365 Defender ATP Audit</label>
+  <fieldset submitButton="false"></fieldset>
+  <row>
+    <panel>
+      <input type="text" token="tkn_host_temp">
+        <label>Host</label>
+        <default></default>
+        <change>
```

Comment:
Use Below instead of <change>:

        <label>Computer Name</label>
         <default>*</default>
         <prefix>*</prefix>
         <suffix>*</suffix>
         <initialValue>*</initialValue>